documentation/service-rules/ecf1/participants.md

```diff
@@ -0,0 +1,567 @@
+
+## User Access
+
+Different users can access participant information based on their role:
+
+### Provider Access
+
+Providers can retrieve participant information through the API, allowing them to:
+- View all participants associated with their partnerships
+- Access detailed information about specific participants
+- Track changes in participant status
+
+### Finance User Access
+
+Finance users can view participant details through the finance application, focusing on:
+- Funding eligibility
+- Pupil premium and sparsity uplifts
+- Mentor funding end dates
+- Cohort information relevant to payments
+
+### Admin User Access
+
+Admin users have comprehensive access through the admin application to:
+- View and manage all participant details
+- Track status changes
+- Monitor partnerships
+- Oversee the entire participant journey
+
+## Error Handling
+
+### Authentication Errors (401 Unauthorized)
+
+If the request does not include valid authentication credentials, the API will return a 401 Unauthorized response:
+
+<pre><code>{
+  "error": "HTTP Token: Access denied"
+}</code></pre>
+
+### Bad Request Errors (400 Bad Request)
+
+If the request contains invalid parameters, the API will return a 400 Bad Request response with details about the errors:
+
+<pre><code>{
+  "errors": [
+    {
+      "title": "Bad request",
+      "detail": "correct json data structure required. See API docs for reference"
+    }
+  ]
+}</code></pre>
+
+### Not Found Errors (404 Not Found)
+
+If the requested participant ID does not exist, the API will return a 404 Not Found response.
+
+## Participant Journeys
+
+### ECT Registration and API Visibility Journey
+
+1. **Registration**: A school induction tutor registers an Early Career Teacher (ECT) in the system, providing their details including name, email, TRN, and school information.
+
+2. **Partnership Check**: The system checks if a partnership exists between the school and a lead provider.
+   - If no partnership exists, the ECT will not be visible via the API.
+   - If a partnership exists, the process continues.
+
+3. **Lead Provider Relationship Check**: The system verifies if the lead provider has a relationship with the school.
+   - If no relationship exists, the ECT will not be visible via the API.
+   - If a relationship exists, the ECT becomes visible via the API.
+
+4. **Data Access**: Once visible via the API, the ECT's data can be:
+   - Retrieved by providers through the API
+   - Accessed by finance users through the finance application
+   - Accessed by admin users through the admin application
+
+5. **Status Changes**: Throughout their journey, an ECT may undergo various status changes:
+   - **Defer**: Training status changes to "Deferred" when the ECT temporarily pauses their training
+   - **Withdraw**: Training status changes to "Withdrawn" when the ECT permanently leaves the programme
+   - **Resume**: Training status changes to "Active" when a previously deferred ECT returns to training
```

Comment:
I may be wrong, but I can't see anything stopping a withdrawn participant from becoming active again?